predicate/solver/ast.py

```diff
@@ -459,36 +459,13 @@ def new(
             + nanoseconds * NANOSECOND
         )
 
-    def __eq__(self, val):
```

Comment:
Is there a chance want to use durations in some other way later?
I'd probably keep these in error in the solver if they are invalid in the current context.

Reply:
Added them back in fb3c45e9a3cdd4307075cd463933cd73dccdd399, throwing an error when an invalid `Option` is created.